ups/daf_butler.table

```diff
@@ -11,3 +11,4 @@ setupRequired(afw)
 
 envPrepend(PATH, ${PRODUCT_DIR}/bin)
 envPrepend(PYTHONPATH, ${PRODUCT_DIR}/python)
+envPrepend(DAF_BUTLER_PLUGINS, lsst.daf.butler.cli.cmd)
```

Comment:
I don't think we want this for butler itself because the butler already knows where its own scripts are. The butler standard set should always work regardless.

For testing I'd create a trivial command line plugin that echoes back the subcommand name. In the test set the environment variable to include the path to that test code and run the new subcommand and check the output.
Note that you can create a class inline in a test and use getFullTypeName on that class and doImport will work on it (since Python will realize it has already been imported). -- the trick is triggering the read of DAF_BUTLER_PLUGINS if some other test has already triggered it.